app/src/androidTest/java/com/github/sdp/tarjetakuna/database/localDatabase/LocalDatabaseProviderTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.github.sdp.tarjetakuna.database.localDatabase
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.sdp.tarjetakuna.database.local.LocalDatabaseProvider
+import com.github.sdp.tarjetakuna.ui.authentication.Authenticator
+import com.github.sdp.tarjetakuna.ui.authentication.SignIn
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import java.util.*
+
+
+@RunWith(AndroidJUnit4::class)
+class LocalDatabaseProviderTest {
+
+    @Before
+    fun setUp() {
+        val mockedAuth = mock(Authenticator::class.java)
+        `when`(mockedAuth.isUserLoggedIn()).thenReturn(true)
+        SignIn.setSignIn(mockedAuth)
+
+        // close the database that could have been opened because of the previous tests
+        LocalDatabaseProvider.closeDatabase("test")

```

Comment:
Given how many times `LocalDatabaseProvider` is present in this file, you can import this statically.